app/sidekiq/evss/disability_compensation_form/upload_bdd_instructions.rb

```diff
@@ -88,20 +114,35 @@ def perform(submission_id)
 
       private
 
+      def file_body
+        @file_body ||= File.read('lib/evss/disability_compensation_form/bdd_instructions.pdf')
+      end
+
       def upload_bdd_instructions
-        client.upload(file_body, document_data)
+        # NOTE: the ApiProviderFactory implements its own Flipper,
+        # ApiProviderFactory::FEATURE_TOGGLE_UPLOAD_BDD_INSTRUCTIONS, which will be used to iteratively
+        # start sending more BDD Instructions to Lighthouse instead of EVSS
+        # However, since using this factory is new behavior, it is hidden behind this additional Flipper, which will act
+        # as a "kill switch" in the event there are problems with the ApiProviderFactory implementation, so we can
+        # revert back to using the EVSS::DocumentsService directly here
+        if Flipper.enabled?(:disability_compensation_use_api_provider_for_bdd_instructions)
+          upload_via_api_provider
+
+          upload_provider.log_upload_success(STATSD_KEY_PREFIX)
+        else
+          EVSS::DocumentsService.new(submission.auth_headers).upload(file_body, document_data)
+        end
       end
 
-      def file_body
-        @file_body ||= File.read('lib/evss/disability_compensation_form/bdd_instructions.pdf')
+      def upload_via_api_provider
+        document = upload_provider.generate_upload_document(
+          BDD_INSTRUCTIONS_DOCUMENT_TYPE, BDD_INSTRUCTIONS_DOCUMENT_TYPE
```

Comment:
Is one of these supposed to be `BDD_INSTRUCTIONS_FILE_NAME`

Reply:
Oops yeah I fixed this on Friday but looks like I forgot to commit and push, fixed and added tests that would have caught this in https://github.com/department-of-veterans-affairs/vets-api/pull/18347/commits/26b2615b6d9430bd44460a82614902d5c2cb6651. Good catch